Sources/AblyAssetTrackingPublisher/State/DefaultSkippedLocationsState.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  Created by Łukasz Szyszkowski on 30/08/2021.
+//
+
+import Foundation
+
+class DefaultSkippedLocatoinsState: PublisherSkippedLocationsState {
+    
+    let maxSkippedLocationsSize: Int
+    private var skippedLocations: [String: [EnhancedLocationUpdate]] = [:]
+    
+    init(maxSkippedLocationsSize: Int = 60) {
+        self.maxSkippedLocationsSize = maxSkippedLocationsSize
+    }
+    
+    func add(trackableId: String, location: EnhancedLocationUpdate) {
+        var locations = skippedLocations[trackableId] ?? []
+        locations.append(location)
+        if locations.count > maxSkippedLocationsSize {
+            locations.remove(at: .zero)
+        }
```

Comment:
Shouldn't we sort locations by time to be sure that we're removing the oldest location update? :thinking:

Reply:
1ad265d7795c5c5ea3680aa014b45b54cf160589